Doc/faq/programming.rst

```diff
@@ -1164,6 +1164,21 @@ This converts the list into a set, thereby removing duplicates, and then back
 into a list.
 
 
+How do you remove multiple items from a list
+--------------------------------------------
+
+As with removing duplicates, explicitly iterating in reverse with a
+delete condition is one possibility.  However, it is easier and faster
+to use slice replacement with an implicit or explicit forward iteration.
+Here are three variations.::
+
+   mylist[:] = filter(keep_function, mylist)
+   mylist[:] = (x for x in mylist if keep_condition)
+   mylist[:] = [x for x in mylist if keep_condition]
```

Comment:
Shouldn't the last 2 be `keep_condition(x)`?

Otherwise looks good.

Reply:
> The conditions are not python function calls, and I prefer not to suggest that they are. I sometimes use to mean code to be substituted, but I don't think that this is established for the FAQ. If some people do a style consistency pass, they can do whatever they decide on.

Ah. I misread the first one as `keep_condition` instead of `keep_function`, so assumed they were all the same.